airbyte-integrations/connectors/source-file/Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM python:3.9-slim
+FROM python:3.9-slim as base
+FROM base as builder
 
-# Bash is installed for more convenient debugging.
-RUN apt-get update && apt-get install -y jq curl bash && rm -rf /var/lib/apt/lists/*
+RUN apt-get update
+WORKDIR /airbyte/integration_code
+COPY setup.py ./
+RUN pip install --prefix=/install .
```

Comment:
Can't we just install this in /usr/local instead of doing the whole multi stage thing?

Reply:
I've emulated [our current generator template](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source-python/Dockerfile).

afaia, the [benefit to multi-staging](https://docs.docker.com/develop/develop-images/multistage-build/) like this is that any unneeded build dependencies, artifacts aren't in the final Docker image, potentially reducing image size (although I haven't tested how much difference it's actually making in the Airbyte case)

Reply:
Oh ok